_1327/main/models.py

```diff
@@ -83,6 +80,15 @@ def can_edit(self, user):
 	def can_delete(self, user):
 		return self.can_edit(user)
 
+	@classmethod
+	def used_permissions(cls):
+		app_label = ContentType.objects.get_for_model(cls).app_label
+		NamedPermission = namedtuple('NamedPermission', ['permission_key', 'description'], verbose=True)
+		return (
+			NamedPermission(permission_key="{app}.{codename}".format(app=app_label, codename=cls.VIEW_PERMISSION_NAME), description=_('view')),
```

Comment:
how about `name` instead of `permission_key`, could you also put the namedtuple into the top level of this module, so that it can be imported from other modules as well?